### PR TITLE
BEE-30969_update_cd_examples

### PR DESCRIPTION
**Pull request by RenatIs**

### CODE REVIEW

**Review by jsmith-cb — DISMISSED** [review]

File: cloudbees-cd/kubernetes/cloudbees-cd-defaults.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ images:
   registry: "docker.io/cloudbees"
 
   # Image tag of the image to pull
-  tag: "10.9.0.157496_3.2.22_20220929"
+  tag: "2023.02.0.160256_3.2.35_20230201"
```
Comment (jsmith-cb) [inline cloudbees-cd/kubernetes/cloudbees-cd-defaults.yaml:10]:
```suggestion
  tag: "2023.03.0.161439_3.2.38_20230307"
```

File: cloudbees-cd/kubernetes/cloudbees-cd-agent-defaults.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 images:
   registry: "docker.io/cloudbees"
   imageRepository: "cbflow-agent"
-  tag: "10.9.0.157496_3.2.22_20220929"
+  tag: "2023.02.0.160256_3.2.35_20230201"
```
Comment (jsmith-cb) [inline cloudbees-cd/kubernetes/cloudbees-cd-agent-defaults.yaml:9]:
```suggestion
  tag: "2023.03.0.161439_3.2.38_20230307"
```

File: cloudbees-cd/kubernetes/values.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ images:
   registry: "docker.io/cloudbees"
 
   # Image tag of the image to pull
-  tag: "10.9.0.157496_3.2.22_20220929"
+  tag: "2023.02.0.160256_3.2.35_20230201"
```
Comment (jsmith-cb) [inline cloudbees-cd/kubernetes/values.yaml:10]:
```suggestion
  tag: "2023.03.0.161439_3.2.38_20230307"
```

**Comment by jsmith-cb** [comment]
@RenatIs or @sachingade20, 
Please verify the changes on [46b48fe](https://github.com/cloudbees/cloudbees-examples/pull/59/commits/46b48fe22fcb402e62e75963e4d4e2fa44539fe5), [4cf5a75](https://github.com/cloudbees/cloudbees-examples/pull/59/commits/4cf5a7536a78b9213899a07da538ee811e883a79), and [19e8abc](https://github.com/cloudbees/cloudbees-examples/pull/59/commits/19e8abcb91d3cc8034d275df7efebd778248c368).

**Review by RenatIs — COMMENTED** [review]
All is ok from my side

**Review by jsmith-cb — APPROVED** [review]

**Pull request merged**